certified-connectors/Azure Machine Learning/Readme.md

```diff
@@ -0,0 +1,2 @@
+### Azure Machine Learning Connector
+Azure Machine Learning provides a comprehensive REST CLI for interacting with Azure Machine Learning Workspaces. This is an attempt to make the use of these APIs easier by exposing these as a connector on the Logic Apps and Power Platforms. This also enables users to leverage these platforms to perform extensive automation tasks without going through the hassle of setting up the CLI by providing UI elements for the same.
```

Comment:
@yashodeepm please format the readme.md file according to the [template](https://github.com/microsoft/PowerPlatformConnectors/blob/dev/templates/certified-connectors/readme.md?plain=1).

Reply:
Made changes